src/api/common.ts

```diff
@@ -27,7 +27,7 @@ export function loadMetrics(): Promise<Array<NodeStatisticalData>> {
 }
 
 export function loadCurrentMetrics(): Promise<Record<string, number>> {
-  return http.get('/monitor_current')
+  return http.get('/monitor_current', { doNotEffectProgress: true })
```

Comment:
`doNotEffectProgress` I think is not a good variable name 😂

Forgive me for saying that.

```suggestion
  return http.get('/monitor_current', { hideTopProgress: true })
```

Reply:
but the progress bar is controlled by a request stack, send a request will trigger stack push, a request received response will trigger stack pop, the progress bar will be in progress when the stack is not empty, so i think the 'hideTopProgress'  maybe can't describe the handler precisely

Reply:
I just think this parameter to describe the action, the action is `hide`, instead of using it to express technical details, maybe I just don't understand the word `effect`. Hmmm.. so, do what you feel is right.

Reply:
even if the quest stack isn't empty when requesting current metrics, will not hide the progress bar, just not trigger stack push and don't trigger stack pop after received metrics ( i think about the variable name long time, don't let me change it, please QAQ

Reply:
how about 'doNotTouchProgress' :P

Reply:
Touch -> Trigger?

Reply:
use it!

Reply:
OK! 😂 Whatever, keep it. It's not important.